packages/contracts/contracts/core/IDAO.sol

```diff
@@ -55,30 +64,47 @@ abstract contract IDAO {
         string calldata _reference
     ) external payable virtual;
 
+    /// @notice Emitted when a deposit is made
+    /// @param sender The address of the sender
+    /// @param token The address of the token deposited
+    /// @param amount The amount of tokens deposited
+    /// @param _reference The deposit reference desribing the reason of it
     event Deposited(
         address indexed sender,
         address indexed token,
         uint256 amount,
         string _reference
     );
-    // ETHDeposited and Deposited are both needed. ETHDeposited makes sure that whoever sends funds
-    // with `send/transfer`, receive function can still be executed without reverting due to gas cost
-    // increases in EIP-2929. To still use `send/transfer`, access list is needed that has the address
-    // of the contract(base contract) that is behind the proxy.
+
+    /**
```

Comment:
Use `///` instead of `*`. 
(This saves two lines of space at the top and bottom)